internal/beater/java_attacher/java_attacher.go

```diff
@@ -160,14 +165,14 @@ func (j *JavaAttacher) Run(ctx context.Context) error {
 			j.logger.Errorf("error during JVMs discovery: %v", err)
 			continue
 		}
-		if err := j.foreachJVM(ctx, jvms, j.attachJVM, true, 30*time.Second); err != nil {
+		if err := j.foreachJVM(ctx, jvms, j.attachJVM, false, 2*time.Minute); err != nil {
```

Comment:
It was changed to be up-to 2 minutes, but not at the intention to increase the interval, only to allow more time before timing out. 

At the moment, the `attachJVM` method blocks until the attacher command exits or is timed out, whichever comes first. 
We may decide that this conceptually not what we want and make it completely asynchronous and not wait for either.
However, if we decide that this is conceptually what we want, the increased timeout should not affect proper attachments, only such that get blocked/hang. I increased that because my tests showed that 30 seconds is realistic even if things work as expected (this may include downloading from maven and verifying pgp signature).

Reply:
OK. Just asking since it seems unrelated to the other changes.